angular-todo/e2e/manage-todo.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+import { v4 as uuid } from 'uuid';
+
+const toDo = uuid() + ' ' + 'TEST';
+const toDoEdited = uuid();
+
+test('manage todo angular', async ({ page }) => {
+  //create new todo
+  await page.goto('https://localhost:8443/home');
+  await page.getByRole('link', { name: 'Sign In', exact: true }).click();
+  await page.getByPlaceholder('User Name').fill('user01');
+  await page.getByPlaceholder('Password').fill('Password1!');
+  await page.getByPlaceholder('Password').press('Enter');
+  await page.getByRole('link', { name: 'Manage your todos here' }).click();
+  await page.getByPlaceholder('What needs doing?').click();
+  await page.getByPlaceholder('What needs doing?').fill(toDo);
+  await page.getByRole('button', { name: 'Create' }).click();
+  await expect(page.getByText(toDo)).toBeVisible();
+
+  //Edit todo
+  await page.locator('li').filter({ hasText: 'Edit Delete' }).locator('div').nth(1).click();
+  await page.getByRole('button', { name: 'Edit' }).click();
+  await page.locator('#editTodo').click();
+  await page.locator('#editTodo').fill(`${toDo} - Edited`);
+  await page.getByRole('button', { name: 'Update Todo' }).click();
+
+  const newTodo = await page.locator('li').locator('div').nth(1).textContent();
```

Comment:
See above

Reply:
Was there a conclusion on this?

Reply:
addressed it on the last commit i send last week

Reply:
done